Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -847,23 +854,28 @@ void doVayDepositionShapeN (const GetParticlePosition& GetPosition,
         const amrex::Real vy = uyp[ip] * invgam;
         const amrex::Real vz = uzp[ip] * invgam;
 
+        // Modify the particle position to match the time of the deposition
+        xp += relative_time * vx;
+        yp += relative_time * vy;
+        zp += relative_time * vz;
```

Comment:
@RemiLehe As pointed out in another comment, doing this preliminary shift in the Esirkepov deposition kernel (instead of combining `relative_time` with `dt` or fractions of `dt` later) makes some checksum benchmarks fail (some tests, both double-precision and single-precision, seem to be particularly sensitive to roundoff changes in that kernel). If that's OK with you, I could maybe show this in a separate PR and we could decide then whether we are OK with rewriting the Esirkepov implementation in a way similar to the Vay deposition and updating the corresponding benchmarks. I would not do this in this PR because it would be less easy to show what is causing the checksum failures.